build-tools/packages/build-tools/src/type-test-generator/main.ts

```diff
@@ -119,10 +142,14 @@ import type * as current from "../../index";
 ];
 
 for (const oldTypeData of previousData) {
+	// Check if the type is from an alpha file
```

Comment:
I don't think we need to do any special alpha handling in here. We'll compare the previous set of types (whichever module(s) they came from) against the current types.